MatrixSDK.xcodeproj/project.pbxproj

```diff
@@ -5509,9 +5509,9 @@
 			buildConfigurationList = 32C6F94019DD814400EA4E9C /* Build configuration list for PBXNativeTarget "MatrixSDK-iOS" */;
 			buildPhases = (
 				385E87177134AE6EBD88F314 /* [CP] Check Pods Manifest.lock */,
+				32C6F92A19DD814400EA4E9C /* Headers */,
```

Comment:
Xcode started complaining about the order of these, but I am actually surprised this ever worked, The headers should be before the sources.